src/Options/Definitions.js

```diff
@@ -898,3 +904,23 @@ module.exports.AuthAdapter = {
     default: true,
   },
 };
+module.exports.LogLevels = {
+  triggerAfter: {
+    env: 'undefinedTRIGGER_AFTER',
```

Comment:
`undefinedTRIGGER_AFTER` something went wrong there, and in the ones below; I think the answer is in the contribution docs

Reply:
Fixed !